qiskit/providers/ibmq/experiment/experimentservice.py

```diff
@@ -335,6 +337,14 @@ def analysis_results(
                 ``analysis_results(quality=[('ge', 'Bad'), ('lt', 'Good')])``
                 will return all analysis results with a quality of ``Bad`` and
                 ``No Information``.
+            tags: Filter by tags assigned to experiments.
```

Comment:
```suggestion
            tags: Filter by tags assigned to analysis results.
```

Looks like copy/paste issue.